gui/common_qt_functions.py

```diff
@@ -1,11 +1,16 @@
+# System imports
+import yaml
+import os
+from sys import platform
+
+# Import QT libraries
 from PySide6 import QtWidgets
 from PySide6.QtWidgets import QWidget
 from PySide6.QtCore import Qt, QSettings
-import yaml
-import os
 
 file_cache_eviction_choices = ['lru','lfu']
 libfusePermissions = [0o777,0o666,0o644,0o444]
+defaultFuseDir = '\Cloudfuse'
```

Comment:
Can you drop the slash here, and just add it when you're building the path?
You could use os.path.join() to build paths without having to hard-code any slashes, which may become important if this variable gets used in Linux at some point.

Reply:
Done